specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_Update.json

```diff
@@ -43,6 +43,9 @@
       }
     },
     "202": {
+      "headers": {
+        "Location": "https://some.endpoint.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/hsm-group/providers/Microsoft.KeyVault/managedHSMs/hsm1?api-version=2022-07-01&kv-operation=abJjb2RkIjoiAGVsZXRlTWFuYWdlZEhzbUFzeW5jYm9"
```

Comment:
Can you please confirm from code to see if RP returns this header in 2020-04-01-preview version? Just to make sure service code and swagger are in sync.

Reply:
Keeping this change as RP does return.